test/Base_Tests/src/Data/Round_Spec.enso

```diff
@@ -176,62 +176,6 @@ add_specs group_builder batch_runner run_advanced =
             round_fun -3.50001 . should_equal -4
             round_fun -3.99999 . should_equal -4
 
-    group_builder.specify "Can round correctly near the precision limit" pending=pending_advanced <|
```

Comment:
Handling these edge cases identically across all backends is either impossible, or only possible by doing it "by hand" and never using built-in primitives. This isn't worth the effort, so these tests aren't necessary at all. The variation in the output for these cases is very small.